DFC.App.ContactUs/Models/EventGridEventData.cs

```diff
@@ -4,6 +4,8 @@ public class EventGridEventData
     {
         public string? Api { get; set; }
 
+        public string? ItemId { get; set; }
```

Comment:
Don't know why the question mark is there.  String is a class so it can be null.